test/helpers.ts

```diff
@@ -2,13 +2,19 @@ import Web3 from "web3";
 import BN from "bn.js";
 import * as _ from "lodash";
 import { Web3Driver } from "../eth";
+import {Driver} from "./driver";
 
 export const retry = (n: number, f: () => Promise<void>) => async  () => {
     for (let i = 0; i < n; i++) {
         await f();
     }
 };
 
+export const evmIncreaseTimeForQueries = async (web3: Web3Driver, seconds: number) => {
+    await evmIncreaseTime(web3, seconds);
+    evmMine(web3, 1); // "bake" the new clock in by closing 1 block
```

Comment:
Why is this needed?

Reply:
Because run query is executed in the context of the previously blosed block. `now` will still be in the past even after we move the clock if a block is not mined to "lock" it it..

Reply:
How peculiar :) Thanks